README.md

```diff
@@ -55,6 +55,7 @@ jobs:
       url: https://pypi.org/p/<your-pypi-project-name>
     permissions:
       id-token: write  # IMPORTANT: this permission is mandatory for trusted publishing
+      # NOTE: setting this permission will remove all other permissions. If you need to do a actions/checkout, for example, you will need to set the "contents" permission to "read"
```

Comment:
The only steps in this job must be downloading the pre-built dists and uploading them. Invoking `git clone` implies that one would perform building right in this job, which is a dangerous idea due to opening up an attack surface for the privilege escalation through dependency poisoning.